packages/reducers/src/makeEnhancer.js

```diff
@@ -8,6 +23,29 @@ import composeReducers from './composeReducers';
 
 export const storeInterface = makeStoreInterface('reducers');
 
+const cleanUpReducer = (state, action) => {
+	if (action.type !== '@redux-tools/REDUCERS_EJECTED') return state;
+
+	const namespaceGroup = action.meta.feature || DEFAULT_FEATURE;
+
+	const pathToNamespace = [namespaceGroup, action.meta.namespace];
+	const pathToReducer = when(always(action.meta.namespace), concat(pathToNamespace))([]);
+
+	const cleanReducers = reduce((state, reducerName) =>
+		dissocPath(append(reducerName, pathToReducer), state)
+	)(__, action.payload);
+
+	const cleanEmptyObjects = compose(
+		when(o(isEmpty, path([namespaceGroup])), dissocPath([namespaceGroup])),
```

Comment:
lens can be used -> u are updating and reading prop from same path

Reply:
maybe creating a new function like Lens => PrevState => NextState would be better? :P